airbyte-integrations/connectors/source-google-adwords-singer/acceptance-test-config.yml

```diff
@@ -0,0 +1,18 @@
+connector_image: airbyte/source-google-adwords-singer:dev
+tests:
+  spec:
+    - spec_path: "source_google_adwords_singer/spec.json"
+  connection:
+    - config_path: "secrets/config.json"
+      status: "succeed"
+    - config_path: "sample_files/invalid_config.json"
+      status: "failed"
+  discovery:
+    - config_path: "secrets/config.json"
+  basic_read:
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "sample_files/configured_catalog.json"
```

Comment:
lets move `invalid_config` and `configured_catalog` to `integration_tests` folder

Reply:
OK